co-authors-plus.php

```diff
@@ -994,24 +994,40 @@ function filter_wp_get_object_terms( $terms, $object_ids, $taxonomies, $args ) {
 	/**
 	 * Filter the count_users_posts() core function to include our correct count
 	 */
-	function filter_count_user_posts( $count, $user_id ) {
+	function filter_count_user_posts( $count = NULL, $user_id, $post_type = NULL ) {
```

Comment:
Is there a reason for marking the first and third arguments as optional? They come from WP Core, so shouldn't they be always set? :)